ssn/chapters/ModelPatterns.html

```diff
@@ -0,0 +1,100 @@
+<!--  <section id="Modeling">  -->
+
+<h2>Common Modeling Questions</h2>
+This section informally discusses how to handle common modeling questions such as locations, forecasts, and
+quantity
+values with a unit of measure.
+
+<section id="location">
+    <h3>Location</h3>
+    <p>Many of the key classes provided by SOSA and SSN represent entities that can be located in space such as
+        sensors, features of interest, actuators, samples, and so forth, or activities that can be located via their
+        participating entities, e.g., platforms.
+        These entities will usually be described using models and ontologies defined for application domains,
+        including
+        technical disciplines, social and business contexts.
+        In these contexts there are a number of implementations that support the expression of spatial properties,
+        including location.
+        These are discussed further in the Spatial Data on the Web Best Practices note [[SDW-BP]]. </p>
+    <p>In particular, GeoSPARQL [[GeoSPARQL]] provides a flexible and relatively complete platform for geospatial
+        objects, that fosters interoperability between geo-datasets. To do so, these entities can be declared as
+        instances of geo:Feature and geometries can be assigned to them via the geo:hasGeometry property.
+        In case of classes, e.g., specific features of interests such as rivers, these can be defined as subclasses
+        of
+        geo:Feature.</p>
+    <p>For example, if observations are made of the atmosphere at a specific location, it might be described as a
+        Sample using the following pattern:</p>
+    <pre class="example" data-include="./rdf/examples/2023ex1.ttl" data-include-format="text"></pre>
```

Comment:
1. The `geo:hasGeometry` property has `rdfs:domain geo:Feature` so this entails that `earthAtmosphere_StE` is of `rdf:type geo:Feature`.  It is not necessary to state it explicitly. And in this case I felt less is more. 

2. Yes, the location could be of the `Sampler`, particularly for in-situ observations. But this example was making a distinct point - that a `Sample` may have a location - using this simple pattern, which allows the location to be capture directly without having to traverse the whole Sampling/Sampler path.

Reply:
1. As it stands, that example graph would produce errors when fed to a reasoner. The specification of the [`geo:hasGeometry` property](https://docs.ogc.org/is/22-047r1/22-047r1.html#_01414b7a-8464-4a9b-897d-46c471274ea6) is strict, its domain and ranges are not meant to be implied.

Broader picture. If GeoSPARQL is to fall within the SDWWG's purview, we better make sure to use it as it is meant to be used. IMHO.

Reply:
> 1. As it stands, that example graph would produce errors when fed to a reasoner. The specification of the [`geo:hasGeometry` property](https://docs.ogc.org/is/22-047r1/22-047r1.html#_01414b7a-8464-4a9b-897d-46c471274ea6) is strict, its domain and ranges are not meant to be implied.

I disagree. As I understand it, the reasoner would not _report errors_, but it would _infer that `earthAtmosphere_StE` is a `geo:Feature`_ ... because of the global `rdfs:domain` constraint. 

The global constraint makes it unnecessary to say `earthAtmosphere_StE a geo:Feature .` since every resource with a `geo:hasGeometry` property is necessarily a `geo:Feature`, which is what the reasoner would infer. 

The function of a reasoner is to add in triples that are implied by the ontology and the data, relieving you of the requirement to include them all in the data explicitly. If any of these triples are inconsistent, we get to dig deeper. In this case, there is nothing inconsistent about `earthAtmosphere_StE a sosa:Sample .` also being inferred to be in the class extension of `geo:Feature`.